lib/src/browser_transport_platform_with_sockjs.dart

```diff
@@ -27,6 +27,13 @@ class BrowserTransportPlatformWithSockJS extends BrowserTransportPlatform
         _sockJSNoCredentials = sockJSNoCredentials == true,
         _sockJSTimeout = sockJSTimeout;
 
+  bool get sockJSDebug => _sockJSDebug;
+  bool get sockJSNoCredentials => _sockJSNoCredentials;
+  List<String> get sockJSProtocolsWhitelist => _sockJSProtocolsWhitelist != null
+      ? new List.from(_sockJSProtocolsWhitelist)
+      : null;
+  Duration get sockJSTimeout => _sockJSTimeout;
```

Comment:
Exposed these for testing, but doesn't hurt to have them public. You only see them if you cast to `BrowserTransportPlatformWithSockJS`